include/net/ethernet.h

```diff
@@ -72,6 +72,9 @@ struct net_eth_addr {
 #endif
 
 #define NET_ETH_MINIMAL_FRAME_SIZE	60
+#define NET_ETH_MAX_DATAGRAM_SIZE	1500
```

Comment:
I would be also for NET_ETH_MTU on that one, it is semantically more relevant

Reply:
Yes, calling it MTU looks actually a bit better here.

Reply:
Changed with filter-edit, hopefully nothing breaks

Reply:
BTW, interesting to look at Linux notion: https://github.com/torvalds/linux/blob/master/include/uapi/linux/if_ether.h#L36